src/ext_traits.rs

```diff
@@ -38,13 +39,15 @@ impl<T: 'static> SelectViewExt for cursive::views::SelectView<T> {
                         match amount {
                             MoveAmount::Extreme => self.set_selection(0),
                             MoveAmount::Integer(amount) => self.select_up(*amount as usize),
+                            _ => Callback::dummy(),  // TODO find a more elegant way to handle this
```

Comment:
This is the only piece of the patch I was less sure about. Is this an appropriate way to ignore the unused `Left/Right` `HalfPage/FullPage` variants?